src/app/static/src/app/store/translations/locales.ts

```diff
@@ -517,6 +541,42 @@ const en: Translations = {
+    privacyUseHeading: "How we use your personal data",
+    privacyUseBody: "We will only use your personal data when the law allows us to.",
+    privacyUseList: "<li>We will need your email address to identify your projects and so we can contact you regarding any submitted troubleshooting requests.</li>" +
+        "            <li>We will use your country input data for fitting the Naomi model.</li>" +
+        "            <li>We may monitor how you use the application in order to improve it.</li>",
+    privacyStoreHeading: "Where we store your data",
+    privacyStoreBody: "We store your data on our own servers based in London, UK",
+    privacySecureHeading: "How we keep your data secure",
+    privacySecureBody: "We have put in place appropriate security measures to prevent your personal data from being accidentally lost, used or accessed in an unauthorised way, altered or disclosed. In addition, we limit access to your personal data to the administrators of Naomi, who have a business need to know, and are subject to a duty of confidentiality.",
+    privacyLengthHeading: "How long will you use my personal data for?",
+    privacyLengthList: "<li>We will retain your personal data indefinitely to fulfil the purposes we collected it for, including for the purposes of satisfying any legal, accounting, or reporting requirements.</li>" +
+        "               <li>We therefore will remove your data when the site is discontinued.</li>" +
+        "               <li>We will remove your data when you request we do so.</li>",
+    privacyDiscloseHeading: "Disclosing your information",
+    privacyDiscloseList: "<li>We may pass on your personal information if we have a legal obligation to do so, or if we have to enforce or apply our terms of use and other agreements.</li>" +
+        "                 <li>We will not share your information with any other organisation for marketing, market research, or any other commercial purposes.</li>",
+    privacyRightsHeading: "Your rights",
+    privacyRightsList: "<li>You can find out what information we hold about you by submitting a subject access request to <a href=\"mailto:naomi-support@imperial.ac.uk\">naomi-support@imperial.ac.uk</a> and ask us not to use the information we collect.</li>" +
+        "               <li>You also have the right to have data erased if it is no longer necessary for the purpose for which it was originally collected/processed, or if there are no overriding legitimate grounds for the processing. This is sometimes known as 'the right to be forgotten'.</li>" +
+        "               <li>To find out more, please see - <a href=\"https://www.imperial.ac.uk/admin-services/secretariat/information-governance/data-protection/internal-guidance/guide-10/\">Guide 12 - Individual Rights | Administration and support services | Imperial College London</a></li>",
```

Comment:
For consistency, I think this should open in another tab (as the other link to non-Naomi Imperial page does).